packages/jest-fake-chalk/.npmignore

```diff
@@ -0,0 +1,4 @@
+**/__mocks__/**
+**/__tests__/**
+src
+yarn.lock
```

Comment:
rm

Reply:
Done (technically we can remove the whole file since this will not be published)